src/rust/src/core/call_manager.rs

```diff
@@ -969,11 +969,12 @@ where
         let mut active_call = check_active_call!(self, "handle_proceed");
         if active_call.call_id() != call_id {
             ringbenchx!(RingBench::CM, RingBench::App, "inactive call_id");
-            return Ok(());
+            Ok(())
+        } else {
+            active_call.set_call_context(app_call_context)?;
+            active_call.inject_proceed(bandwidth_mode)
         }
 
```

Comment:
Also, can you remove this empty line?

You could run this to fix the format:
```
cd src/rust
./scripts/format-code
```

See the script for the nightly build that is currently needed.

Also, there are changes in the next release which will run GitHub Actions to see formatting and clippy issues for PRs, so you would know right away if there are any issues like this.

Thanks!

Reply:
done!